File: eBiblioteka/eBiblioteka.Core/Entities/ToDo210923.cs

namespace eBiblioteka.Core
{
    public class ToDo210923 : BaseEntity
    {
        public string ActivityName { get; set; } = null!;
        public string ActivityDescription { get; set; }=null!;
        public DateTime FinshingDate { get; set; }
        public string StatusCode { get; set; }=null !;

        public int UserId { get; set; }
        public User User { get; set; } = null!;
    }
}

File: eBiblioteka/UI/mobile_ebiblioteka/lib/models/todo210923.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:mobile_ebiblioteka/models/user.dart';

part 'todo210923.g.dart';

@JsonSerializable()
class ToDo210923 {
  int? id;
  String? activityName;
  String? activityDescription;
  DateTime? finshingDate;
  String? statusCode;
  int? userId;
  User? user;

  ToDo210923(this.id, this.activityName, this.activityDescription,
      this.finshingDate, this.statusCode, this.userId, this.user);

  factory ToDo210923.fromJson(Map<String, dynamic> json) =>
      _$ToDo210923FromJson(json);

  Map<String, dynamic> toJson() => _$ToDo210923ToJson(this);
}
